order/buyer/cancel.md

```diff
@@ -11,7 +11,14 @@ Cancel an order line in Tradecloud when it is cancelled at the buyer.
 * `Cancelled` lines cannot be cancelled again
 
 {% hint style="warning" %}
-When using the single delivery per order line feature, the `cancelled` indicator is only supported for the first order line of each item number. The other lines with the same item, prices and terms will also be cancelled together with the first line.
+**Single delivery order line cancellation behavior:**
+
+When using the single delivery per order line feature:
+
+1. The `cancelled` indicator should only be set on the **primary order line** (the first line with a specific item number)
+2. Tradecloud will automatically cancel all related split lines that reference the primary line via their `originalPosition` field.
+
+This ensures consistent cancellation across all related order lines without requiring individual cancellation of each split line.
```

Comment:
This is not correct anymore since the release of https://tradecloud.atlassian.net/browse/TC-9388

Single lines that are cancelled are left out of the delivery schedule.